src/promptflow-core/promptflow/executor/flow_executor.py

```diff
@@ -214,9 +215,13 @@ def create(
             setup_exporter_from_environ()
 
         if hasattr(flow_file, "__call__") or inspect.isfunction(flow_file):
+            from ._prompty_executor import PromptyExecutor
             from ._script_executor import ScriptExecutor
 
-            return ScriptExecutor(flow_file, connections=connections, storage=storage)
+            if isinstance(flow_file, Prompty):
```

Comment:
Maybe we can do like this?
if isinstance(flow_file, Prompty)
    return
if hasattr()
    return

Reply:
fix it